### DISCUSSION TITLE
visual studio 2022 intellicode not working for winui3

### DISCUSSION THREAD

**Discussion by ifgotmoney** [microsoft/WindowsAppSDK#3888] [Q&A]
visual studio 2022 intellicode not working for winui3 for XAML documents
does anyone have the same issue with VS2022 community+WinUI3(C++)

VS2019 works fine with C++/WinUI3.

**Comment by naughtykid001** [comment]
Interesting, something breaks in WinUI3? VS2022 should be miles away from 2019 preview version.
But it seems C++ doesn't really get any excitements from developer community.
